disk/disk_windows.go

```diff
@@ -218,14 +218,17 @@ func IOCountersWithContext(ctx context.Context, names ...string) (map[string]IOC
 			if err != nil {
 				return drivemap, err
 			}
-			drivemap[path] = IOCountersStat{
-				ReadBytes:  uint64(diskPerformance.BytesRead),
-				WriteBytes: uint64(diskPerformance.BytesWritten),
-				ReadCount:  uint64(diskPerformance.ReadCount),
-				WriteCount: uint64(diskPerformance.WriteCount),
-				ReadTime:   uint64(diskPerformance.ReadTime / 10000 / 1000), // convert to ms: https://github.com/giampaolo/psutil/issues/1012
-				WriteTime:  uint64(diskPerformance.WriteTime / 10000 / 1000),
-				Name:       path,
+
+			if len(names) == 0 || common.StringsHas(names, path) {
```

Comment:
If `names` is empty slice, I think `common.StringsHas` always return `false`. So no need to check `len(names) == 0`.

```suggestion
			if common.StringsHas(names, path) {
```

Reply:
Hi @shirou , thanks for the comment, but my understanding is the if clause should return `true` rather than `false` if `names` is empty. In other words, the logic will skip filtering by `names` and return all disks when  `names` is empty. This is also how the linux version and darwin version is implemented today:
- Darwin: https://github.com/shirou/gopsutil/blob/master/disk/disk_darwin_cgo.go#L38
- Linux: https://github.com/shirou/gopsutil/blob/master/disk/disk_linux.go#L409

Reply:
Ah, I apologize. I was mistaken (reviewing when tired is not a good idea). This feature is a "filter", so if there are no conditions, it should add everything. You are indeed correct.